### ISSUE TITLE
Led blinking sequence not increasing, stuck?

### ISSUE THREAD

**Issue by deadlyknapsack** [marcone/teslausb#397]
I'm trying to install the latest stable release on my Pi Zero W with a Sandisk 128GB sd card. Image is etched fine and I changed the .conf accordingly. When I power the Pi with the sd card in it, it blinks twice and then the led shows it's processing. Led turns off, blinks twice and does it again. Seems like it is doing it in loops and not stopping It's not visible in my network, can't ssh to it, I don't see the Access Point on the Pi show up and I'm not sure what to do...

Any tips?

**Comment by marcone** [comment]
Double-check your wifi information. If you think it correct, put the sd card in a PC and look at the teslausb-headless-setup.log file on the boot drive.

**Comment by deadlyknapsack** [comment]
I triple checked the wifi, it is good. 

Also, I don't see any logfiles, at all. When I let it sit for half an hour (seems like it is going off an on again), nothing on boot is changed. There is supposed to be some kind of marker file for the setup right?

Just to be sure, Balena says the drive is unusually big for a drive. Is that just a warning, to make sure you don't accidentally flash a SSD or something or might 128gb be to big for teslausb?

**Comment by marcone** [comment]
> I triple checked the wifi, it is good.
> 
> Also, I don't see any logfiles, at all. When I let it sit for half an hour (seems like it is going off an on again), nothing on boot is changed. There is supposed to be some kind of marker file for the setup right?

There should be several, like TESLAUSB_SETUP_STARTED, WIFI_ENABLED, TESLAUSB_SETUP_FINISHED

If you're not even seeing /boot/teslausb-headless-setup.log though, then something else is wrong because creating that file is pretty much the first setup does.

> Just to be sure, Balena says the drive is unusually big for a drive. Is that just a warning, to make sure you don't accidentally flash a SSD or something or might 128gb be to big for teslausb?

That's just an Etcher warning to prevent you from accidentally flashing the boot drive of your computer. Teslausb has no limit on drive size.

**Comment by deadlyknapsack** [comment]
I don't know what to tell you... just unwrapped a new Sandisk and new Raspberry Pi Zero W... I could try a different SD, but I only have one Zero laying around. Not common for these things to be dead on arrival right?

I just did another fresh etch and fresh .conf upload. Letting it run for 15 minutes, gonna unplug it and see if it is working. It's not showing up on the network either. It's 2.4 and my iPhone and Macbook Pro are able to connect to the wifi network without issues.

**Comment by marcone** [comment]
When you plug the sd card in to a PC, what files do you see in the "boot" drive?
What is the content of the /boot/cmdline.txt and /boot/config.txt files?

**Comment by deadlyknapsack** [comment]
Exactly the same as right after etching.


cmdline.txt:
```
dwc_otg.lpm_enable=0 console=serial0,115200 console=tty1 root=PARTUUID=22eddddb-02 rootfstype=ext4 elevator=deadline fsck.repair=yes rootwait modules-load=dwc2,g_ether quiet
```
config.txt
```
# For more options and information see
# http://rpf.io/configtxt
# Some settings may impact device functionality. See link above for details

# uncomment if you get no picture on HDMI for a default "safe" mode
#hdmi_safe=1

# uncomment this if your display has a black border of unused pixels visible
# and your display can output without overscan
#disable_overscan=1

# uncomment the following to adjust overscan. Use positive numbers if console
# goes off screen, and negative if there is too much border
#overscan_left=16
#overscan_right=16
#overscan_top=16
#overscan_bottom=16

# uncomment to force a console size. By default it will be display's size minus
# overscan.
#framebuffer_width=1280
#framebuffer_height=720

# uncomment if hdmi display is not detected and composite is being output
#hdmi_force_hotplug=1

# uncomment to force a specific HDMI mode (this will force VGA)
#hdmi_group=1
#hdmi_mode=1

# uncomment to force a HDMI mode rather than DVI. This can make audio work in
# DMT (computer monitor) modes
#hdmi_drive=2

# uncomment to increase signal to HDMI, if you have interference, blanking, or
# no display
#config_hdmi_boost=4

# uncomment for composite PAL
#sdtv_mode=2

#uncomment to overclock the arm. 700 MHz is the default.
#arm_freq=800

# Uncomment some or all of these to enable the optional hardware interfaces
#dtparam=i2c_arm=on
#dtparam=i2s=on
#dtparam=spi=on

# Uncomment this to enable the lirc-rpi module
#dtoverlay=lirc-rpi

# Additional overlays and parameters are documented /boot/overlays/README

# Enable audio (loads snd_bcm2835)
dtparam=audio=on
dtoverlay=dwc2
```
list of directory and files:
```
overlays
bcm2708-rpi-0-w.dtb
bcm2708-rpi-b-plus.dtb
bcm2708-rpi-b.dtb
bcm2708-rpi-cm.dtb
bcm2708-rpi-zero-w.dtb
bcm2708-rpi-zero.dtb
bcm2709-rpi-2-b.dtb
bcm2710-rpi-3-b-plus.dtb
bcm2710-rpi-3-b.dtb
bcm2710-rpi-cm3.dtb
bootcode.bin
cmdline.txt
config.txt
COPYING.linux
fixup_cd.dat
fixup_db.dat
fixup_x.dat
fixup.dat
issue.txt
kernel.img
kernel7.img
LICENCE.broadcom
LICENSE.oracle
ssh
start_cd.elf
start_db.elf
start_x.elf
start.elf
teslausb_setup_variables.conf
teslausb_setup_variables.conf.sample
wpa_supplicant.conf.sample
```
teslausb_setup_variables.conf
```
#####################################################################
# SAMPLE CONFIGURATION FILE FOR TESLAUSB Pi Setup
#
# Example config file for teslausb_setup. Lines with "#" are comments/ignored.
# Remove the "#" before "export" to activate a line. Be sure to rename this file
# to "teslausb_setup_variables.conf" and place it in the "boot" folder of your
# SD card.
# Please check the latest version of this file at:
# https://github.com/marcone/teslausb/blob/main-dev/pi-gen-sources/00-teslausb-tweaks/files/teslausb_setup_variables.conf.sample
#
######################################################################

# Default variables for CIFS (Windows/Mac file sharing) setup
export ARCHIVE_SYSTEM=cifs
export archiveserver=your_archive_name_or_ip
export sharename=your_archive_share_name
export shareuser=username
export sharepassword=password
# export cifs_version="3.0"

# Uncomment the following line and select drive if you want to use an external drive instead of built-in MicroSD card.
# External USB drive is supported only for RPi4.
# export usb_drive=/dev/sda

# Uncomment to disable snapshots. This also disables samba and RecentClips-extension
#export SNAPSHOTS_ENABLED=false

# Recommendations on image sizes:
# Notes:
#   * You can use relative sizes to the available space (e.g., 10%) or explicit sizes (e.g., 16G)
#   * The /backingfiles partition holds both the camera and music disk images.
#   * Please note that the setup script will reserve a fixed 6GB of free space regardless of sd card size.
# When using snapshots:
#    Using images that fill the entire available space will make snapshots inoperable.
#    The system removes snapshots when the available space on the /backingfiles partition falls
#    below the size of the camera image.
#    * camsize should be a fairly small explicit value to leave room for the snapshots.
#      The recommended size is between 8GB and 16GB.
#    * musicsize should also be an explicit size  but should be large enough to hold your music
#      library (obviously).
# When not using snapshots:
#    It is usually easiest to specify an explicit size for your music drive based on your actual
#    library plus an expected growth factory. You can then set the camera size to a portion of the
#    remaining space.
#    * camsize should still be a fairly small explicit value. The size is determined by the number of
#      sentry mode and saved clips you expect to have before the RPi can offload them.
#      * A single SentryClips/SavedClips event is about 1.2GB (MCU2.5) to 1.5GB (FSD) of data.
#      * The hour of RecentClips is about 6.9GB (MCU2.5) to 8.8GB (FSD) of data.
#        The car will delete clips from the RecentClips that are older than an hour.
#    * musicsize can be an explicit or relative size but should be large enough to hold your music
#      library (obviously).
export camsize=100G
#export musicsize=4G

# If you want to automatically copy music from a CIFS share every time
# the Pi connects to wifi, set the following variable. The share is
# assumed to exist on the same server as the archive share. It can
# be the same share as the share used for backing up recordings, but
# the folder needs to be different.
# export musicsharename=your_music_share/music_folder

# Wifi setup information. Note that Raspberry Pi Zero W only supports 2.4 GHz wifi
export SSID='teslausb'
export WIFIPASS='teslausb'

# If doing a headless (i.e. automatic) setup
export HEADLESS_SETUP=true

# uncomment to enable a samba server that exports the recordings
#export SAMBA_ENABLED=true
# uncomment to enable guest access to the samba server
#export SAMBA_GUEST=true

# uncomment to enable teslausb to act as a wifi access point with the given
# SSID, so you can access it while on the road.
#export AP_SSID='TESLAUSB WIFI'
# Change this! The setup script will not accept the default 'password'
# Also note that the wifi password must be at least 8 characters.
#export AP_PASS='password'
# IP address is optional. The AP will give its clients IP addresses in the
# x.x.x.100-150 range so make sure you don't put the AP's IP address in
# that range.
#export AP_IP='192.168.66.1'

# uncomment if you want to set the time zone to something other than the default 'BST'
# Can be an actual timezone, or "auto" to attempt automatic timezone detection
# export timezone="America/Los_Angeles"

# By default there is a 20 second delay between connecting to wifi and
# starting the archiving of recorded clips. Uncomment this to change
# the duration of that delay
# export archivedelay=20

# Uncomment if you want to override the default hostname of "teslausb"
# export TESLAUSB_HOSTNAME=teslausb-Model3

# Uncomment if setting up Pushover push notifications
# export pushover_enabled=false
# export pushover_user_key=user_key
# export pushover_app_key=app_key

# Uncomment if setting up Gotify push notifications
# export gotify_enabled=false
# export gotify_domain=https://gotify.domain.com
# export gotify_app_token=put_your_token_here
# export gotify_priority=5

# Uncomment if setting up IFTTT push notifications
# export ifttt_enabled=false
# export ifttt_event_name=put_your_event_name_here
# export ifttt_key=put_your_key_here

# Uncomment if setting up AWS SNS notifications
# export sns_enabled=true
# export aws_region=us-east-1
# export aws_access_key_id=put_your_accesskeyid_here
# export aws_secret_key=put_your_secretkey_here
# export aws_sns_topic_arn=put_your_sns_topicarn_here

# TeslaUSB can optionally use the Tesla API to keep your car awake, so it can
# power the Pi long enough for the archiving process to complete. To enable
# that, please provide your Tesla account email and password below.
# TeslaUSB will only send your credentials to the Tesla API itself.
#
# NOTE:  if you do this, it's highly recommended that you set up 'PIN to Drive'
#   as if the Pi or the car is stolen, the thief has access to your Tesla account
#   and could change the password, add their own devices as a key, steal the car,
#   etc.  If either is stolen, CHANGE YOUR TESLA PASSWORD IMMEDIATELY.
#
# export tesla_email=joeshmo@gmail.com
# export tesla_password=teslapass
# Please also provide your vehicle's VIN, so TeslaUSB can keep the correct
# vehicle awake.
# export tesla_vin=5YJ3E1EA4JF000001

# Uncomment if you want to increase the size of the root
# filesystem so there's extra space for installing additional
# software. This only works if the backing store partition hasn't
# been created yet, i.e. during early setup, and requires an extra
# reboot. Sizes can be specified as for example "500M" or "2G"
# export INCREASE_ROOT_SIZE=500M

# Uncomment and change if you want setup scripts to be pulled
# from a different repo than github.com/marcone/teslausb
# export REPO=marcone

# Uncomment and change if you want a different branch than main-dev
# export BRANCH=main-dev
```

**Comment by mobiletru** [comment]
Did you format sd card correctly ?

**Comment by marcone** [comment]
>```
> ######################################################################
> 
> # Default variables for CIFS (Windows/Mac file sharing) setup
> export ARCHIVE_SYSTEM=cifs
> export archiveserver=your_archive_name_or_ip
> export sharename=your_archive_share_name
> export shareuser=username
> export sharepassword=password
>```

Just in case this is the actual conf file you're using: the 4 variables above should all be specified. I assume your archive server isn't actually called `your_archive_name_or_ip`

> ```
> export SSID='teslausb'
> export WIFIPASS='teslausb'
>```

Similarly here, this should be the name and password of the wifi network that the Pi is going to connect, e.g. your home wifi. Or did you actually set up a separate wifi access point for the Pi to connect to, with name `teslausb` and password `teslausb` ?

**Comment by deadlyknapsack** [comment]
> > ```
> > ######################################################################
> > 
> > # Default variables for CIFS (Windows/Mac file sharing) setup
> > export ARCHIVE_SYSTEM=cifs
> > export archiveserver=your_archive_name_or_ip
> > export sharename=your_archive_share_name
> > export shareuser=username
> > export sharepassword=password
> > ```
> 
> 
> Just in case this is the actual conf file you're using: the 4 variables above should all be specified. I assume your archive server isn't actually called `your_archive_name_or_ip`

Right. I assumed I could skip this, as I intent to only use the AP of the Pi on the go. My Tesla is not in range of my wifi unfortunately. Will it prevent setup? I could temporarily setup a share.

> 
> 
> > ```
> > export SSID='teslausb'
> > export WIFIPASS='teslausb'
> > ```
> 
> 
> Similarly here, this should be the name and password of the wifi network that the Pi is going to connect, e.g. your home wifi. Or did you actually set up a separate wifi access point for the Pi to connect to, with name `teslausb` and password `teslausb` ?

Yes, I did. Just to make sure that my wifi wasn't trying to force the client on the 5Ghz.

**Comment by marcone** [comment]
> > > ```
> > > ######################################################################
> > > 
> > > # Default variables for CIFS (Windows/Mac file sharing) setup
> > > export ARCHIVE_SYSTEM=cifs
> > > export archiveserver=your_archive_name_or_ip
> > > export sharename=your_archive_share_name
> > > export shareuser=username
> > > export sharepassword=password
> > > ```
> > 
> > 
> > Just in case this is the actual conf file you're using: the 4 variables above should all be specified. I assume your archive server isn't actually called `your_archive_name_or_ip`
> 
> Right. I assumed I could skip this, as I intent to only use the AP of the Pi on the go. My Tesla is not in range of my wifi unfortunately. Will it prevent setup? I could temporarily setup a share.

The archive not being reachable will certainly cause setup to fail. It should fail much later than what you're apparently seeing though, and log the failure in /boot/teslausb-headless-setup.log.
If you don't want it to archive, you can set `ARCHIVE_SYSTEM` to `none`. However, your conf file is also missing the variables needed for specifying the Pi's own AP information (`AP_SSID` and `AP_PASS` at a minimum, and you might want `SAMBA_ENABLED` too)

> > > ```
> > > export SSID='teslausb'
> > > export WIFIPASS='teslausb'
> > > ```
> > 
> > 
> > Similarly here, this should be the name and password of the wifi network that the Pi is going to connect, e.g. your home wifi. Or did you actually set up a separate wifi access point for the Pi to connect to, with name `teslausb` and password `teslausb` ?
> 
> Yes, I did. Just to make sure that my wifi wasn't trying to force the client on the 5Ghz.

And you're sure this 'teslausb' wifi is working properly?

Which image did you flash?

**Comment by deadlyknapsack** [comment]
> > > > ```
> > > > ######################################################################
> > > > 
> > > > # Default variables for CIFS (Windows/Mac file sharing) setup
> > > > export ARCHIVE_SYSTEM=cifs
> > > > export archiveserver=your_archive_name_or_ip
> > > > export sharename=your_archive_share_name
> > > > export shareuser=username
> > > > export sharepassword=password
> > > > ```
> > > 
> > > 
> > > Just in case this is the actual conf file you're using: the 4 variables above should all be specified. I assume your archive server isn't actually called `your_archive_name_or_ip`
> > 
> > 
> > Right. I assumed I could skip this, as I intent to only use the AP of the Pi on the go. My Tesla is not in range of my wifi unfortunately. Will it prevent setup? I could temporarily setup a share.
> 
> 
> The archive not being reachable will certainly cause setup to fail. It should fail much later than what you're apparently seeing though, and log the failure in /boot/teslausb-headless-setup.log.

I'm trying again with the settings set to a share on my Macbook Pro.

> If you don't want it to archive, you can set `ARCHIVE_SYSTEM` to `none`. However, your conf file is also missing the variables needed for specifying the Pi's own AP information (`AP_SSID` and `AP_PASS` at a minimum, and you might want `SAMBA_ENABLED` too)

I figured after a few tries I should just disable the AP option for now and just get it working as simply set up as possible.

> 
> > > > ```
> > > > export SSID='teslausb'
> > > > export WIFIPASS='teslausb'
> > > > ```
> > > 
> > > 
> > > Similarly here, this should be the name and password of the wifi network that the Pi is going to connect, e.g. your home wifi. Or did you actually set up a separate wifi access point for the Pi to connect to, with name `teslausb` and password `teslausb` ?
> > 
> > 
> > Yes, I did. Just to make sure that my wifi wasn't trying to force the client on the 5Ghz.
> 
> 
> And you're sure this 'teslausb' wifi is working properly?
> 
> 
> Which image did you flash?

Yes, 100% working fine. Both Macbook Pro and iPhone are running on it. 
I'm using this one: teslausb-20190615

**Comment by mobiletru** [comment]
https://www.youtube.com/watch?v=ETs6r1vKTO8   
Have you watched this video

**Comment by deadlyknapsack** [comment]
> https://www.youtube.com/watch?v=ETs6r1vKTO8
> Have you watched this video

Absolutely. I'm doing exactly what he is doing. It just fails the moment I put the SD in the Pi. It looks like it is in some infinite loop. I just reformatted it again, using terminal to be sure. Etched it again and still only flashes twice, following by some pulsating blink (processing) and two blinks and again and again and again. 

When I power off, put the SD in my mac, nothing has changed on the SD.

**Comment by mobiletru** [comment]
Find a pc with a sd card reader and watch that video 10 times

**Comment by deadlyknapsack** [comment]
> Find a pc with a sd card reader and watch that video 10 times

I'm not an idiot. I have 3 other Pi's running in my household. I don't need to watch a 30-minute video 10 times for this to get to work. I really appreciate the help, but that is a really unhelpful remark.

**Comment by mobiletru** [comment]
export ARCHIVE_SYSTEM=cifs
export archiveserver=your_archive_name_or_ip
export sharename=your_archive_share_name
export shareuser=username
export sharepassword=password

your not watching the video

**Comment by deadlyknapsack** [comment]
> export ARCHIVE_SYSTEM=cifs
> export archiveserver=your_archive_name_or_ip
> export sharename=your_archive_share_name
> export shareuser=username
> export sharepassword=password
> 
> your not watching the video

Can you go "help" somebody else? Thanks.

**Comment by deadlyknapsack** [comment]
@marcone I just did everything you suggested, but still the Pi keeps looping with a freshly etched SD, updated .conf. After 10-15 minutes, still nothing has changed on the SD card. Could it be a hardware issue? Actual broken Pi?

**Comment by mobiletru** [comment]
export ARCHIVE_SYSTEM=none  
 you didn't do everything he suggested

**Comment by marcone** [comment]
I have no idea. The flashes you're seeing are likely not the flashes that the teslausb setup process would show, since it would have created the log file if it got that far, and you're not seeing the log file, so it's stopping very early in the boot process.
Could be a defective Pi, or inadequate power to the Pi.
One more thing you can try is to not put a config file at all, i.e. just flash the image and then stick it in the Pi and turn it on. This will make setup end early, but it should leave a USB network interface configured, which should allow you to ssh in to the Pi.

**Comment by deadlyknapsack** [comment]
Alright, couple of tries later, I switched the USB cord and I'm getting somewhat further. The log is now being created, but it fails:

```
Mon 13 Jan 18:17:33 GMT 2020 : Detecting whether to update wpa_supplicant.conf
Mon 13 Jan 18:17:33 GMT 2020 : Wifi variables specified, and no /boot/WIFI_ENABLED. Building wpa_supplicant.conf.
Mon 13 Jan 18:17:33 GMT 2020 : Rebooting...
Mon 13 Jan 18:18:01 GMT 2020 : Detecting whether to update wpa_supplicant.conf
Mon 13 Jan 18:18:01 GMT 2020 : Grabbing main setup file.
Mon 13 Jan 18:18:22 GMT 2020 : get_script failed, retrying
Mon 13 Jan 18:18:46 GMT 2020 : get_script failed, retrying
Mon 13 Jan 18:19:09 GMT 2020 : get_script failed, retrying
Mon 13 Jan 18:19:33 GMT 2020 : get_script failed, retrying
```

I read something about the local time being off, because NTP wouldn't sync. I changed the time from June 15th to current date manually through SSH, because none of the Googled ntp fix suggestions worked for me. Is my ISP blocking something that sets the time correctly? 

Even with manual time set to - somewhat accurate, talking milliseconds off) - it fails the get_script. 

Any suggestions? 

EDIT: I know 100% sure that the network is working, the wifi details I put in .conf are 100% correct. I just don't get it. It's not showing up in my DHCP list either...

EDIT 2: I tried copying my wifi credentials in wpa_supplicant.conf and during booting, it picks it up, deletes the file (makes sense I guess), but still it is not connected to the wifi network.

**Comment by marcone** [comment]
I'm not sure I understand your comments. You say that your Pi is not connected to wifi, but you're also asking if your ISP is blocking something. If your Pi is not on your wifi, then whatever your ISP does is irrelevant. First step should be to get your Pi onto wifi. That's not a teslausb problem though.

**Comment by deadlyknapsack** [comment]
I agree. My edit/additions to my comment are just new findings. At first I thought it was an NTP issue, because the get_script was failing (as I was reading about other issues), then I saw it wasn't connected to wifi in the first place and I tried a different approach. I just thought I updated my comment. I'm still trying and I do realise it's not a teslausb problem.

**Comment by mobiletru** [comment]
Your wifi is not 2.4 ghz keep it simple

**Comment by marcone** [comment]
@mobiletru you're not helping. If you read above you'll see that @deadlyknapsack specifically created a 2.4GHz-only network.

@deadlyknapsack are you sure you actually have a Pi Zero W, and not a Pi Zero? You wouldn't be the first person to accidentally order the wrong one (been there myself). To rule out any issue with the teslausb image, you might consider installing a standard Raspbian desktop image and see how that goes. If that still doesn't work, I'd say you have a defective Pi.

**Comment by deadlyknapsack** [comment]
@marcone 100% Pi Zero W. I double checked the print on the PCB. Also, I specifically ordered it, because the stack of normal Zero's I have laying around aren't compatible. I was already planning on installing Raspbian and see how the Wifi connection goes. Thanks for your help.

**Comment by henroFall** [comment]
Make sure to use a utility to force the card to a fat32 format? I've
recently had problems with 128 gb cards when I forget that step. I had a
habit of just blowing all partitions off the card, formatting exFAT , then
let Win32DiskImager make the card. But that quit working and my pi would
boot like you describe. The SD card would get super hot, too. Like can't
touch it hot.

Once I formatted the card as fat32 that all quit.

Did you specially format the card first ?

**Comment by marcone** [comment]
@henroFall Honestly, that sounds kinda bogus to me. If you flash a Raspbian image onto the card (either one of the teslausb prebuilts or a standard Raspbian one), that effectively formats the card by overwriting the partition table and creating two new partitions. I don't see what formatting it first would do, unless you do a "slow" format where it writes every sector of the card, which might allow it to find and remap bad blocks.

**Comment by henroFall** [comment]
I know, right? I've been working with Pis for years and never did anything
to preformat the SD cards first. But the other day I was stuck hard and the
only thing that fixed it was using guiformat to format the card. I agree,
it blows all of that out and makes new partitions, anyway, so I don't get
it. But it worked, and it's easy to try/eliminate so I thought I'd suggest
it.

My symptoms were pretty much the same as what was described here or I
wouldn't have brought it up.

**Comment by deadlyknapsack** [comment]
@marcone I flashed Buster Lite, created a wpa_supplicant.conf, booted it in the same pi and watched it through HDMI boot nicely and at the end it was online. I could ssh into it and run "apt-get update". It was properly online. So, Pi is working, wifi is working. 

I then formatted the SD card, etched your image, replaced the teslausb_setup_variables.conf with my own and copied the working (from the Buster Lite test) wpa_supplicant.conf onto the sd card. I then booted it, with HDMI connected and I could see it go through the first boot up steps, I could see the IP it was getting from my router. Then, it reboots, does a bunch of stuff and guess what... it still fails to get the get_script. It looks like it connects to wifi just fine the first boot run, then on the second one it says something like: "checking to see if wpa_supplicant.conf needs to be updated" and after that it doesn't work... it just tries the get_script infinitely... also, can't reach it over ssh like I could with the Buster boot.

EDIT: When I open wpa_supplicant.bak I can see it changed the password to "TEMPPASS". Is that correct?

**Comment by deadlyknapsack** [comment]
@marcone I did the manual installation on a Buster Lite images and it went flawlessly. I honestly think a step in your auto installation is/was causing it. It must be the part where you deal with the wpa_supplicant.conf details. As I said above, first boot up makes your image go on the wifi shortly, second boot up fails. But with Buster Lite it just connect every single boot up... I'm happy it works, but it would still bother me if I was the developer haha. Thanks for the help.

**Comment by deadlyknapsack** [comment]
@marcone still have one question, if you don't mind. Now that I did the manual installation and it is working in my Tesla, I would like it to act as an AP. In your description, you mention to add the AP details to the teslausb_setup_variables.conf, but where is that file stored when I do a manual installation?

If I follow this, would that be enough: https://www.raspberrypi.org/documentation/configuration/wireless/access-point.md

**Comment by deadlyknapsack** [comment]
@marcone Sorry for spamming, but I couldn't let it go... I actually got it to work. I'd like to share my findings, because there might be something in it, worth to you:

When I was looking through the logs on my tv, while it was attached to HDMI, I noticed something like:

```[Starting] Disable Wifi if country isn't defined```

When I was making a `wpa_supplicant.conf` for the Buster Lite tryout, I remember using an example `wpa_supplicant.conf` which started with `country=US`. When I etched your latest release again (for the gazillionth time), I created a file next to `wpa_supplicant.conf.sample` called `wpa_supplicant.conf`and put this into it:

```
country=NL
ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
update_config=1

network={
  ssid="my_ssid_name"
  scan_ssid=1
  psk="my_wifi_password"
  key_mgmt=WPA-PSK
}
```

When I booted the pi with the teslausb sd in it, it connected to wifi fine, where it didn't when I just filled out the wifi details in the `teslausb_setup_variables.conf`

Does that make sense? With the `country`set to `NL` the `get_script`wasn't failing like it used to for literally dozens of tries.
